strider/compatibility.py

```diff
@@ -252,11 +252,11 @@ def map_curie(
             categories, identifiers = data[curie]
         except KeyError:
             return [curie]
-        prefixes = [
+        prefixes = {
             prefix
             for category in categories
             for prefix in prefixes.get(category, [])
-        ]
+        }
```

Comment:
Not exactly sure why this should be a set instead of a list. Am I missing something here?

Reply:
It doesn't need to be ordered, there should be no duplicates, and we'd like to be able to search it quickly.